R/pkgs/report_generation/R/ReportBuildUtils.R

```diff
@@ -1,1112 +1,1679 @@
+                                    pop_name,
+                                    pdeath_level = "high", ## doesn't really matter since data should be the same for infections
+                                    scenario_label,
+                                    scenario_col,
+                                    sim_start_date,
+                                    sim_end_date,
+                                    plot_intervention = FALSE, ## may not want to plot if it is too complicated
+                                    interv_start_date = NA,
+                                    interv_end_date = NA) {
+
+  geopop <- geodata %>%
+    dplyr::filter(include_in_report) %>%
+    dplyr::select(!! pop_name) %>%
+    unlist %>% unname
+
+  ##TODO: Make this so each scenario does not use the same sims...though should not matter.
+  to_plt <- hosp_state_totals %>%
+    dplyr::filter(pdeath==pdeath_level) %>%
+    dplyr::mutate(scenario_name = factor(scenario_name,
+                                         levels = scenario_label,
+                                         labels = scenario_label)) %>%
+    dplyr::filter(scenario_name == scenario_label) %>%
+    dplyr::group_by(time) %>%
+    dplyr::summarise(NincidInf_med = median(NincidInf), NincidInf_mean = mean(NincidInf), NincidInf_lo = quantile(NincidInf, .25), NincidInf_hi = quantile(NincidInf, .75)) %>%
+    dplyr::mutate(pop = sum(geopop)) %>%
+    dplyr::mutate(AR_med = NincidInf_med/pop*100000,
+                  AR_mean = NincidInf_mean/pop*100000,
+                  AR_lo = NincidInf_lo/pop*100000,
+                  AR_hi = NincidInf_hi/pop*100000)
+
+  rc <- ggplot(data=to_plt, aes(x = time)) +
+    geom_pointrange(aes(y = AR_med, ymin = AR_lo, ymax = AR_hi), colour = scenario_col) +
+    scale_y_continuous("Infections per 100K", labels = scales::comma) +
+    scale_x_date(date_breaks = "1 month",
+                 date_labels = "%b",
+                 limits = c(lubridate::ymd(sim_start_date), lubridate::ymd(sim_end_date))) +
+    theme_minimal() +
+    theme(axis.title.x =  element_blank(),
+          legend.position = "bottom",
+          legend.title = element_blank())
+
+
+  if(plot_intervention){
+    interv_dates <- data.frame(xmin = lubridate::ymd(interv_start_date),
+                               xmax = lubridate::ymd(interv_end_date),
+                               ymin = 0,
+                               ymax = 1.05*max(to_plt$AR_hi))
+
+    rc <- rc +
+      geom_rect(data = interv_dates, inherit.aes = FALSE,
+                aes(xmin = xmin, xmax = xmax, ymin = ymin, ymax = ymax),
+                color = "grey", fill = "grey", alpha = 0.33)
+  }
+
+  return(rc)
+
+}
+
+
+##'
+##' Plot modeling assumption parameter distributions
+##'
+##' @param name parameter name
+##' @param config config file
+##' @return plot of distribution of peak timing across simulations by county
+##'
+##' @export
+##'
+plot_model_parameter_distributions <- function(name, config){
```

Comment:
I never got `plot_model_parameter_distributions` to work and I'm not  sure this should go into master.

Reply:
This should be resolved by the PR